metrics/web/lib/dashboard/presentation/widgets/build_result_bar_graph.dart

```diff
@@ -69,36 +69,44 @@ class _BuildResultBarGraphState extends State<BuildResultBarGraph> {
       buildResults: _barsData,
     );
 
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.end,
-      children: <Widget>[
-        Flexible(
+    return Column(
```

Comment:
It seems like the `Align` widget with the alignment `bottomCenter` would be enough for now as we don't introduce the date range.
```suggestion
    return Align(
```